vm/src/value.rs

```diff
@@ -539,16 +540,23 @@ pub struct ValuePrinter<'a> {
     pub value: Variants<'a>,
     pub max_level: i32,
     pub width: usize,
+    pub debug_level: &'a DebugLevel,

```

Comment:
Any reason why this is taken by value? It is fine either way I think but by value seems slightly simpler.

Reply:
Ok, just wondering!